src/pages/home/report/ReportActionsView.js

```diff
@@ -174,53 +176,79 @@ class ReportActionsView extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        const isReportFullyVisible = this.getIsReportFullyVisible();
+
+        // NETWORK CHANGE DETECTED
+        // When returning from offline to online state we want to trigger a request to OpenReport which
+        // will fetch the reportActions data and mark the report as read. If the report is not fully visible
+        // then we call ReconnectToReport which only loads the reportActions data without marking the report as read.
         if (lodashGet(prevProps.network, 'isOffline') && !lodashGet(this.props.network, 'isOffline')) {
-            if (this.getIsReportFullyVisible()) {
+            if (isReportFullyVisible) {
                 Report.openReport(this.props.report.reportID);
             } else {
                 Report.reconnect(this.props.report.reportID);
             }
         }
 
-        // The last sequenceNumber of the same report has changed.
         const previousLastSequenceNumber = lodashGet(CollectionUtils.lastItem(prevProps.reportActions), 'sequenceNumber');
         const currentLastSequenceNumber = lodashGet(CollectionUtils.lastItem(this.props.reportActions), 'sequenceNumber');
-
-        // Record the max action when window is visible and the sidebar is not covering the report view on a small screen
-        const isReportFullyVisible = this.getIsReportFullyVisible();
-        const sidebarClosed = prevProps.isDrawerOpen && !this.props.isDrawerOpen;
         const screenSizeIncreased = prevProps.isSmallScreenWidth && !this.props.isSmallScreenWidth;
-        const reportBecomeVisible = sidebarClosed || screenSizeIncreased;
 
+        // NEW REPORT ACTION APPEARED
         if (previousLastSequenceNumber !== currentLastSequenceNumber) {
             const lastAction = CollectionUtils.lastItem(this.props.reportActions);
             const isLastActionFromCurrentUser = lodashGet(lastAction, 'actorEmail', '') === lodashGet(this.props.session, 'email', '');
+
+            // NEW REPORT ACTION APPEARED FROM CURRENT USER
             if (isLastActionFromCurrentUser) {
-                // If a new comment is added and it's from the current user scroll to the bottom otherwise leave the user positioned where they are now in the list.
+                // If a new comment is added and it's from the current user scroll to the bottom otherwise leave the user positioned where
+                // they are now in the list.
```

Comment:
I think this comment should be above `if (isLastActionFromCurrentUser) {` and replace the all caps one